### PR TITLE
EditAndDeletePost

### PR DESCRIPTION
**Pull request by themikail**

### CODE REVIEW

File: pages/components/cards.js [hunk 1]
```diff
@@ -195,3 +278,37 @@ const LikeIconImage = styled.img`
...
+
+const DropdownItem = styled.button`
+  background-color: #fff;
+  border-width: 0px;
+  padding: 12px 16px;
+  display: block;
+  cursor: pointer;
+  width: 70%;
+  margin-left: 35px;
+`;
```
Comment (doemser) [inline pages/components/cards.js:306]: This button is called item which is misleading. Better create one styled button that can handle both of the variants here

File: pages/components/cards.js [hunk 2]
```diff
@@ -195,3 +278,37 @@ const LikeIconImage = styled.img`
...
+
+const DropdownButton = styled.button`
+  background-color: transparent;
+  margin-left: 10px;
+  padding: 8px;
+  border: none;
+  cursor: pointer;
+`;
+
+const DropdownContent = styled.div`
```
Comment (doemser) [inline pages/components/cards.js:290]: can you find a better html element than div?

File: pages/components/cards.js [hunk 3]
```diff
@@ -195,3 +278,37 @@ const LikeIconImage = styled.img`
...
+const DropdownItem = styled.button`
+  background-color: #fff;
+  border-width: 0px;
+  padding: 12px 16px;
+  display: block;
+  cursor: pointer;
+  width: 70%;
+  margin-left: 35px;
+`;
+const DropdownContainer = styled.div`
```
Comment (doemser) [inline pages/components/cards.js:307]: same here, isnt there a better element than a div?

**Review by doemser — CHANGES_REQUESTED** [review]
In your "post lovely music" form, under "Another songs" you have a heading that does nothing and a button that you can't click. Please remember that we should not have anything in our app that does not work.

Also I left you some comments on your code you should fix before merging.

File: pages/components/cards.js (hunk 3 above)
Comment (themikail) [reply pages/components/cards.js:307]: Change to UL

Comment (themikail) [reply pages/components/cards.js:297]: change to ul

File: pages/components/cards.js (hunk 2 above)
Comment (themikail) [reply pages/components/cards.js:280]: change to li

**Review by NikRoe — CHANGES_REQUESTED** [review]
Hi there,

this looks very nice and promising. 

However, there is still one suggestion left from a previous review that you should implement:
You can make your `DropdownButton` even more reusable by incorporating what is currently part of your `DropdownItem`.
This also solves the naming issue where the DropdownItem is a button.

Keep it up 🙌

**Pull request merged**